zenoh-protocol/tests/channel.rs

```diff
@@ -326,3 +347,76 @@ fn channel_tcp_udp() {
         close_session(router_manager, client_session, locators).await;
     });
 }
+
+#[cfg(all(feature = "unix", target_family = "unix"))]
+#[test]
+fn channel_tcp_unix() {
+    let _ = std::fs::remove_file("/tmp/socket_1.sock");
+    // Define the locator
+    let locators: Vec<Locator> = vec![
+        "tcp/127.0.0.1:7449".parse().unwrap(),
+        "unix//tmp/socket_1.sock".parse().unwrap(),
```

Comment:
Change the file name to something more self-explanatory, e.g., zenoh-test-unix-socket_1.sock